VERSION.cmake

```diff
@@ -1,6 +1,6 @@
 SET(VERSION_MAJOR "2")
 SET(VERSION_MINOR "46")
-SET(VERSION_PATCH "30")
+SET(VERSION_PATCH "31")
```

Comment:
well, version:bump do common default action, which is increase of patch level. But not all changes fir into patch number increase. I worry we do not change rake version:bump, as detection if API or ABI or none change is too difficult.